extensions/interactions/MusicNotesInput/directives/MusicNotesInputRulesService.ts

```diff
@@ -38,12 +38,12 @@ oppia.factory('MusicNotesInputRulesService', [
           _convertSequenceToMidi(inputs.x));
       },
       IsLongerThan: function(answer, inputs) {
-        return _convertSequenceToMidi(answer).length > inputs.x;
+        return _convertSequenceToMidi(answer).length > inputs.k;
```

Comment:
Hi @seanlip, since I added test for the `HasLengthInclusivelyBetween` function, I also thought of adding tests of `IsLongerThan` since only these two functions did not have tests in the spec file. After looking into the function, I did not find this comparison correct as this was always returning false since inputs.x is not an integer field but a list instead.

The format for inputs.x is:
```
x: [{
          readableNoteName: 'A4',
          noteDuration: {
            num: 1,
            den: 1
          }
        }]
```

I tried to see the exploration yaml by creating an downloading an exploration but it did not work due to #6931.

So, I checked the file `pitch_perfect.yaml` where the rule of isLongerThan requires inputs.k. You can check [here](https://github.com/oppia/oppia/blob/develop/data/explorations/pitch_perfect.yaml#L823).

I am not a lot familiar with this interaction. So, can you please confirm if this change is correct. Thanks!

Reply:
Yes, you are correct. Thank you very much for checking and for doing this!